INSTALL

```diff
@@ -90,8 +90,10 @@ Further information about the required dependencies can be found here:
 **pyshp** 2.0 or later (https://pypi.python.org/pypi/pyshp)
     Pure Python read/write support for ESRI Shapefile format.
 
-**PROJ** 4.9.0 or later (https://proj4.org/)
-    Cartographic Projections library.
+**PROJ** 8.0.0 or later (https://proj4.org/).
+
+**pyproj** 3.0.0 or later (https://github.com/pyproj4/pyproj/)
+    Python interface to PROJ (cartographic projections and coordinate transformations library).
```

Comment:
Do we need to say we're depending on Proj if we use pyproj now?

Reply:
In the current state, the geodesic stuff relies on PROJ. https://github.com/SciTools/cartopy/pull/1808#issuecomment-890398217